apps/condo/package.json

```diff
@@ -48,12 +48,14 @@
     "bull": "^3.22.0",
     "bull-repl": "^0.26.3",
     "classnames": "^2.2.6",
+    "connect-redis": "^6.0.0",
     "date-fns": "^2.20.1",
     "dd-trace": "^0.32.0",
     "emotion": "^10.0.27",
     "emotion-server": "^10.0.27",
     "esdk-obs-nodejs": "3.21.3",
     "express": "^4.17.1",
+    "express-session": "^1.17.2",
```

Comment:
if you put the logic inside @core/keystone you should move the requirenebts inside `packages/@core.keystone/packages.json`